File: Python/room.py
class Room:
    def __init__(self, room_number, room_type, price_per_night, max_occupancy, amenities):
        self.room_number = room_number
        self.is_occupied = False
        self.room_type = room_type
        self.price_per_night = price_per_night
        self.max_occupancy = max_occupancy
        self.amenities = amenities

    def get_room_type(self):
        return self.room_type

    def get_room_number(self):
        return self.room_number

    def get_price_per_night(self):
        return self.price_per_night

    def get_max_occupancy(self):
        return self.max_occupancy

    def get_amenities(self):
        return self.amenities

File: C++/Room.cpp
#include "Room.h"

Room::Room(int roomNumber, const std::string &roomType, double pricePerNight, int maxOccupancy, const std::string &amenities)
    : roomNumber(roomNumber), isOccupied(false), roomType(roomType), pricePerNight(pricePerNight), maxOccupancy(maxOccupancy), amenities(amenities)
{
}

std::string Room::getRoomType()
{
    return roomType;
}

int Room::getRoomNumber() const
{
    return roomNumber;
}

double Room::getPricePerNight() const
{
    return pricePerNight;
}

int Room::getMaxOccupancy()
{
    return maxOccupancy;
}

std::string Room::getAmenities()
{
    return amenities;
}
